cmd/ir2proxy/main.go

```diff
@@ -53,7 +53,10 @@ func run() int {
 			return 1
 		}
 
-		hp, translationErrors := translator.IngressRouteToHTTPProxy(ir)
+		hp, translationErrors, err := translator.IngressRouteToHTTPProxy(ir)
+		if err != nil {
+			log.Fatal(err)
+		}
 		if len(translationErrors) > 0 {
 			if hp == nil {
```

Comment:
I think we discussed that unless err != nil, there will always be a np returned, in that case can this logic be simplified to

hp, warnings, err := translate(...)
for _, w := range warnings {
log.Warn(w)
}

Reply:
We did discuss that, and I forgot. Sorry.